DEVELOPER.md

```diff
@@ -1,8 +1,10 @@
 ### Instruction for launching tests of Model Analyzer on Ubuntu
 
+> NOTE: The Model Analyzer branches are synced with branches in the [OpenVINO](https://github.com/openvinotoolkit/openvino) repositories
```

Comment:
> NOTE: The Model Analyzer branches are synced with branches in the [OpenVINO](https://github.com/openvinotoolkit/openvino) repositories.
(why repositories not repository?)
It is recommended to use Python virtual environment to work with the repository. (without Note)